specs/specTemplate/sampleComponentSpec.md

```diff
@@ -109,14 +167,31 @@ Detailed description of how input behaves. Logic, corner cases, patterns, etc.
 
 
 
-### Touch
+#### **Touch**
 
 | Event | Action              | NOTE                    |
 | ----- | ------------------- | ----------------------- |
 | tap   | what happens on tap | Side note (if relevant) |
 | drag  |                     |                         |
 
 
+
+## RTL
+
+> We are deciding on how are we going to handle the RTL. Detailed description will be added later.
+
+
+
+## DOM structure
+
+Each component should have a visual style guide for all of its visual states and elements structure. This style guide will be based on the **Style API**, and a visual theme agreed upon as our reset style (Wix style).
+
```

Comment:
ALL of the dom structure should be explained here?

Sounds like an overkill. The developer should be able to decide that. Just make sure that the things that are described in the Style API are there